common/src/main/java/org/apache/comet/parquet/Native.java

```diff
@@ -24,6 +24,13 @@
 import org.apache.comet.NativeBase;
 
 public final class Native extends NativeBase {
+
+  static {
```

Comment:
Any reason for moving this from the base class?

Reply:
It turns out that we still rely on the static initializer for the native Parquet code, which I had not realized when I started on this PR.

Reply:
Hmm, if only scan is enabled, can we still catch the exception whiling unable to load the native lib?

Reply:
We could potentially catch `UnsatisfiedLinkError` (and other exceptions) in `NativeBase.bundleLoadLibrary` or `NativeBase.load`. But this being static initialization, I'm not sure if the logger would have been initialized at that point.